src/styles/partials/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  &__nav__item {
```

Comment:
This will resolve to `nav__list__nav__item`

Reply:
Yup, @SviatoslavVasyletskyi that's why `font-weight: 500;` is not applied